content/c/concepts/strings/terms/strcmp/strcmp.md

```diff
@@ -15,7 +15,7 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-The `strcmp()` string function compares two strings and returns an integer value.
+The **`strcmp()`** string function compares two strings and returns an integer value.
```

Comment:
Can we apply a similar edit to the `Description` on line # 3? (e.g., "Compares two strings and returns an integer value.")